src/twisted/test/test_tcp.py

```diff
@@ -1363,15 +1363,21 @@ def test_buildProtocolClient(self):
 
 
 class LargeBufferWriterProtocol(protocol.Protocol):
-
     # Win32 sockets cannot handle single huge chunks of bytes.  Write one
     # massive string to make sure Twisted deals with this fact.
 
     def connectionMade(self):
         # write 60MB
-        self.transport.write(b'X'*self.factory.len)
-        self.factory.done = 1
-        self.transport.loseConnection()
+        self.transport.write(b'X'*(self.factory.len-1))
+
+        def finish():   # write last byte separately to check issue #9446
```

Comment:
I guess that the inline command can be set as a docstring for this function.
Rather than "check issue #9446", maybe this command can describe the targeted behaviour regardless of the ticket number.

Reply:
commited change